triagebot.toml

```diff
@@ -383,7 +383,7 @@ message_on_add = """\
 - Priority?
 - Regression?
 - Notify people/groups?
-- Needs `I-nominated`?
+- Needs `I-{team}-nominated`?
```

Comment:
I'm curious how the `{team}` placeholder is resolved (probably by the triagebot?) :thinking: 

I'll check that, too.

Reply:
Oh right, `{variable}` gets substituted by triagebot. Well, I don't know how triagebot will deal with this, I don't think it will substitute a team. `{team}` was only meant as a placeholder for humans (“some team”) not for triagebot. If you have concrete suggestions I change the placeholder.

Reply:
I think when the Triagebot is about to emit the notification to Zulip ([in this point](https://github.com/rust-lang/triagebot/blob/92c3399ac77706649c2026279f124d4e61e6d899/src/handlers/notify_zulip.rs#L134)) if there is a `T-*` label associated with the issue I can infer which team to use to replace `{team}`.

Not 100% it works as I suspect, I need to check a few assumptions but worth a try. Let me get back to you on this before merging this patch :slightly_smiling_face:

Reply:
I can submit a patch to triagebot too if you'd like to. It should be as simple as:

```rs
let mut teams = event
    .issue
    .labels
    .iter()
    .map(|label| &label.name)
    .filter_map(|label| label.strip_prefix("T-"));
if let Some(team) = teams.next()
    && teams.next().is_none()
{
    topic = topic.replace("{team}", team);
}
```

However, it would hard-code the concept of “teams” and rust-lang/rust's labeling system (in which <kbd>T-*</kbd> labels represent teams) and as far as I remember, triagebot wants to stay repository-agnostic.

Reply:
@fmease left a proposal at https://github.com/rust-lang/triagebot/pull/1766